content/22.vaccines.md

```diff
@@ -237,6 +237,14 @@ Furthermore, the producers of Gam-COVID-Vac state that they can produce the vacc
 ### Inactivated Whole Virus Vaccines
 
 <!--Rephrase to match figure? Or rephrase figure?-->
+This type of vaccine uses a "killed" full viral particle to prime the immune system.
+The virus is killed using formaldehyde or $beta$-propiolactone [@doi:10.1080/14760584.2017.1357471].
```

Comment:
```suggestion
The virus is killed using formaldehyde or &beta;-propiolactone [@doi:10.1080/14760584.2017.1357471].
```
We've had issues with Greek letters not rendering at various points in the manuscript building process, so I'm going to suggest sticking with the one I know works even though I'm sure there are probably other ways to get there as well 😆